test/functional/page_objects/visual_builder_page.ts

```diff
@@ -64,6 +64,19 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
       }
     }
 
+    public async checkTabIsSelected(chartType: string) {
+      const chartTypeBtn = await testSubjects.find(`${chartType}TsvbTypeBtn`);
+      const isSelected = await chartTypeBtn.getAttribute('aria-selected');
```

Comment:
are you sure that **chartTypeBtn.getAttribute('aria-selected')** returns boolean value for your case?  If I'm not wrong it return string representation 'false'/'true'

right example of usage 
![image](https://user-images.githubusercontent.com/20072247/91487929-312e5280-e8b7-11ea-8964-c4397fc5cb79.png)

Reply:
Good catch!
Thanks, updated!